migration/sql-files/002-fix-273-workitemtype.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE work_item_types
+SET fields = '{"system.state": {"Type": {"Kind": "enum", "Values": ["new", "in progress", "resolved", "closed"], "BaseType": {"Kind": "string"}}, "Required": true}, "system.title": {"Type": {"Kind": "string"}, "Required": true}, "system.creator": {"Type": {"Kind": "user"}, "Required": true}, "system.assignee": {"Type": {"Kind": "user"}, "Required": false}, "system.description": {"Type": {"Kind": "string"}, "Required": false}}';
```

Comment:
I don't think you imply the above, I think you meant something like:

```
wit, err := witr.Load(ctx, typeName)
// convert wit to app.WorkItemType
// update fields 
// do the convertFieldTypeToModels for the fields' map.
// create a WorkItemType
db.Model(&updatedWit)
```

Let me if this is the idea you mentioned, @aslakknutsen .